demos/action_recognition/python/README.md

```diff
@@ -1,6 +1,6 @@
 # Action Recognition Python\* Demo
 
-![](./action_recognition.gif)
+![](../action_recognition.gif)
```

Comment:
Having gifs for demos which have only one version of implementation in a folder with README has a nice effect: GitHub collapses folders. For example `action_recognition/python` will be viewed as a single folder. We should exploit it.

Reply:
Done